apps/api-gateway/src/chat/chat.controller.ts

```diff
@@ -1,9 +1,14 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Post, Body } from '@nestjs/common';
 import { ApiTags } from '@nestjs/swagger';
 import { ChatService } from './chat.service';
 
 @Controller('chat')
 @ApiTags('Chat API')
 export class ChatController {
   public constructor(private readonly chatService: ChatService) {}
+
+  @Post('create-room')
+  public createChatRoom(@Body() createRoomDto: { roomName: string }) {
+    return this.chatService.createRoom(createRoomDto.roomName);
+  }
```

Comment:
```suggestion
  @Post('create-room')
  public createChatRoom(@Body() createRoomDto: CreateRoomReqDto) {
    return this.chatService.createRoom(createRoomDto.roomName);
  }
```

위와 같은 방식으로 DTO 를 만들어서 관리해도 될 거 같아요!

Reply:
[010f01e](https://github.com/GrebbMe/grabbme-be/pull/35/commits/010f01e17674c56b5634489a039a74f9f3c15a23)

위 커밋에 반영하였습니다!